feat(readme): Update (#205)
Clinical-Genomics/trailblazer: README.md

### Monitor the progress of analysis workflows submitted to SLURM

Trailblazer is a tool that aims to provide:

> - Monitoring of processes requiring submission to a workflow manager

Reviewer: ```suggestion
- Monitoring of processes that require submission to a workflow manager
```

> - Display metadata for each analysis in webinterface

Reviewer: ```suggestion
- Display metadata for each analysis within a web based user interface
```

[Here][Trailblazer-UI] you can find a simple web UI for Trailblazer that helps you keep track of the status of multiple runs

## Installation

> Trailblazer written in Python 3.7+ and is available on the [Python Package Index][pypi] (PyPI).

Reviewer: ```suggestion
Trailblazer is written in Python 3.7+ and is available on the [Python Package Index][pypi] (PyPI).
```

pip install --editable .
```

> Files will be verified as blacked automatically with each push to GitHub. If you would like to automatically [Black] format your commits on your local machine:

Reviewer: ```suggestion
With each push to GitHub your files will be automatically verified using [Black] . If you would like to automatically [Black] format your commits on your local machine:
```

```
pre-commit install
```

## Contributing

> Trailblazer is using GitHub flow branching model as described in Atlas [GitHub Flow].

Reviewer: ```suggestion
Trailblazer uses the GitHub flow branching model as described in Atlas [GitHub Flow].
```

## Documentation

Here's a brief documentation. Trailblazer functionality can be accessed from the command line interface (CLI), the monitoring web interface, the supporting REST API, as well as using the Python API.

### Command line interface

#### Command: `trailblazer init`

> Setup (or reset) a Trailblazer database. It will simply set up all the tables in the database. You can reset an existing database by using the `--reset` option.

Reviewer: ```suggestion
Setup (or reset) a Trailblazer database. The command will set up all the tables in the database. You can reset an existing database by using the `--reset` option.
```
